js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,90 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options,
+    });
+
+    this.settings = app.settings.clone( );
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toJSON()));
+  }
+
+  get events() {
+    return {
+      'change input[name="windowControlStyle"]': 'changedWindowStyle'
+    };
+  }
+
+  changedWindowStyle( event ) {
+    if ( event.target.id == 'windowControlStyleMac' ) 
```

Comment:
Do we really want to update the model (and therefore UI) as soon as the user changes the field? The rest of the form is persisted when you click save. It's a bit odd UX for me that localSettings are updated as soon as you change them in the form whereas server persisted settings happen when you click save.

The fact that one saves to localStorage whereas the other to the server, IMO, shouldn't change the UX. It's an implementation detail.

I could live with it if you guys feel otherwise, but it feels weird to me.

@morebrownies what do you think?